lib/fruitbot/streampusher_api.ex

```diff
@@ -92,7 +100,20 @@ defmodule Fruitbot.StreampusherApi do
     "Next show is #{title}, hosted by #{host}! Description: #{description}. :link: #{url}"
   end
 
-  # def today do
-  #
-  # end
+  @spec commands() :: String.t()
+  def commands do
+    """
+    !vr
+    !donate
+    !advice
+    !sorry
+    !thisisamazing
+    !gohackyourself
+    !next
+    !wiki
+    !tag
+    !datafruiter
+    !commands
+    """
```

Comment:
Looks good! 
Maybe it would make more sense if this goes in the commands module since it's technically not part of the streampusher api.

Reply:
Oh right! For sure. I'll do the drag and drop